libtrip-notification/src/main/res/values-fr/strings.xml

```diff
@@ -1,17 +1,14 @@
 <resources>
     <!-- Notification Strings -->
-    <string name="end_navigation">Fin de navigation</string>
-    <string name="kilometers">km</string>
-    <string name="meters">m</string>
-    <string name="miles">mi</string>
-    <string name="feet">ft</string>
```

Comment:
unused resources, it's duplicated with `libnavigation-util` resources

Reply:
Great catch! That was stale from the previous refactor 👍